bigquery/indexer.py

```diff
@@ -240,13 +243,56 @@ def read_table(client, table_name):
         client.dataset(dataset_id, project=project_id).table(table_name))
 
 
+def create_samples_json_export_file(es, index_name, project_id):
+    """
+    Writes the samples export JSON file to a GCS bucket. This significantly 
+    speeds up exporting the samples table to Saturn in the Data Explorer.
+
+    Args:
+        es: Elasticsearch object.
+        index_name: Name of Elasticsearch index.
+        project_id: Google Cloud Project ID containing the export samples bucket
+    """
+    entities = []
+    search = Search(using=es, index=index_name)
+    for hit in search.scan():
+        participant_id = hit.meta['id']
+        doc = hit.to_dict()
+        for sample in doc.get('samples', []):
+            sample_id = sample['sample_id']
+            export_sample = {'participant': participant_id}
+            for table_column, value in sample.iteritems():
+                # Ignore _has_* and sample_id fields.
```

Comment:
(It took me a while to understand this)

table_column -> es_field_name
```
                # es_field_name looks like "_has_chr_18_vcf", "sample_id" or "verily-public-data.human_genome_variants.1000_genomes_sample_info.In_Low_Coverage_Pilot".
                splits = table_column.split('.')
                # Ignore _has_* and sample_id fields.
                if len(splits) != 4:
```

Reply:
Done.